packages/core/src/infrastructure/bull/bull.types.ts

```diff
@@ -617,6 +618,19 @@ export const StudentBullJob = z.discriminatedUnion('name', [
   }),
 ]);
 
+export const ResumeReviewBullJob = z.discriminatedUnion('name', [
```

Comment:
do we need a discriminated union here since we only have a single object?

Reply:
Nice catch! Simplified the schema